src/server.js

```diff
@@ -259,20 +259,20 @@ class Server {
     const events = this._events
     return Promise.map(tasks.keySeq(), (queue) => {
       return this._rabbitmq.subscribeToQueue(queue, (job, done) => {
-        this._runWorker(queue, tasks.get(queue), job, done)
+        return this._runWorker(queue, tasks.get(queue), job, done)
       })
     })
-      .then(() => {
-        return Promise.map(events.keySeq(), (exchange) => {
-          return this._rabbitmq.subscribeToFanoutExchange(
-            exchange,
-            (job, done) => {
-              this._runWorker(exchange, events.get(exchange), job, done)
-            }
-          )
-        })
+    .then(() => {
```

Comment:
yea, I say if it passes standard lets not argue since to me having it under is more readable vs a break. It makes me think it is not part of the above chain.